src/components/calScreen.js

```diff
@@ -1,11 +1,18 @@
 import React from 'react';
+import { PropTypes } from 'prop-types';
 
-export default function CalScreen() {
+export default function CalScreen({ total, operation, next }) {
   return (
     <>
       <div className="display-btn">
-        <span>0</span>
+        <span>{total || operation || next}</span>
       </div>
     </>
   );
 }
+
+CalScreen.propTypes = {
+  total: PropTypes.oneOf([null]).isRequired,
+  operation: PropTypes.oneOf([null]).isRequired,
+  next: PropTypes.oneOf([null]).isRequired,
+};
```

Comment:
- [ ] You have done great so far. :+1: :+1: Looking at the video below, you can see that your prop validation has an error, `Failed prop type: Invalid prop `next` of value `93698` supplied to `CalScreen`, expected one of [null].`.  you can consider setting the values of those props to empty string instead of null and then add a `PropTypes.string.isRequired,`. I'll suggest you look into this. :+1: 

- [ ] Also when I enter a number and then an operator, the operator doesn't show up on the screen. I'll suggest you look into this too. :wave: 

- VIDEO REFERENCE:

https://github.com/manzitresor/math_magicians/assets/107632780/8f74fae9-5987-4262-80b1-aaad3715819d

Reply:
- Well resolved 👍🏽 
<img width="1297" alt="Screenshot 2023-07-05 at 17 35 52" src="https://github.com/manzitresor/math_magicians/assets/58692590/2dfad463-9eae-4201-a3eb-c2060770ac2b">